tests/autodiff/generic-impl-jvp.slang

```diff
@@ -8,8 +8,8 @@ typedef float Real;
 
 typealias IDFloat = IFloat & IDifferentiable;
 
-__generic<T, let N : int>
-struct dvector
+__generic<T : IDifferentiable, let N : int>
+struct dvector : IDifferentiable
```

Comment:
I'm not totally sure why `dvector` needs to be `IDifferentiable`? This test doesn't do higher order differentiation, and without `IDifferentiable`, wouldn't `dvector` be treated as conforming to `IDifferentiable` via `DifferentialBottom`?

Reply:
I think I just forgot to remove it.